jenkins/ci.suse.de/templates/cloud-crowbar-pipeline-template.yaml

```diff
@@ -170,9 +170,9 @@
           multi-select-delimiter: ','
           # NOTE: keeping designate and octavia disabled by default, until their
           # implementation is complete
-          default-value: '{enabled_services|database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
+          default-value: '{enabled_services|designate,database,rabbitmq,keystone,swift,monasca,glance,cinder,neutron,nova,horizon,ceilometer,heat,manila,trove,barbican,magnum,sahara,aodh,tempest}'
```

Comment:
enabling designate by default will impact all Crowbar ECP jobs. we shouldn't do it unless we know for a fact that it deploys successfully and doesn't cause any problems at runtime

Reply:
... and that includes running tempest

Reply:
this PR can be merged after https://github.com/crowbar/crowbar-openstack/pull/2282 and https://github.com/crowbar/crowbar-openstack/pull/2294 merged .